### PR TITLE
latent etc

### PR DESCRIPTION
**Pull request by cscherrer**
Fixes https://github.com/cscherrer/MeasureBase.jl/issues/70

### CODE REVIEW

File: src/latent-joint.jl [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+"""
+    jointof(m)
+
+$_LATENT_DOCSTRING
+"""
+function jointof(m)
+    fwd(x) = x => x
+    
+    function back(p::Pair)
```
Comment (oschulz) [inline src/latent-joint.jl:55]: Isn't `back` a bit too generic?

Comment (cscherrer) [reply src/latent-joint.jl:55]: Yes, but it's only in scope in a very limited context

Comment (oschulz) [reply src/latent-joint.jl:55]: Sure, why not then.

**Comment by oschulz** [comment]
Fine with me!

**Pull request merged**